client/index.html

```diff
@@ -3,10 +3,18 @@
   <head>
     <meta charset="UTF-8" />
     <link rel="icon" type="image/svg+xml" href="/favicon.png" />
+    <link rel="stylesheet" type="text/css" href="/client/src/style.css" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
   </head>
   <body>
     <div class="content" style="position: absolute; width: 100vw; height: 100vh; top: 0; left: 0; overflow: hidden; background: black; margin: 0; padding: 0;"></div>
-    <script type="text/javascript" src="https://cdn.jsdelivr.net/npm/itk-vtk-viewer@11.12.4/dist/itkVtkViewer.js"></script>
+    <div id="loadingText" class="loadingText">
+      TensorBoard 3D is loading, please wait...
+    </div>
+    <div id="loading" class="loading"/>
+    <div id="progress" class="progress"/>
+    <script type="text/javascript" src="https://cdn.jsdelivr.net/npm/itk-vtk-viewer@11.12.4/dist/itkVtkViewer.js">
```

Comment:
to make loading faster, can this go in `<head>`, maybe with `defer` or `async`? https://flaviocopes.com/javascript-async-defer/

Reply:
Sure thing! I moved the script into the `<head>` using defer, thanks for the recommendation.